w3-account.md

```diff
@@ -12,61 +12,93 @@
-2. Recovering access if the user loses access to all devices is also a challenge.
+Account can be used to solve both discovery and recovery problems:
 
-To address these issues, we propose the concept of an account as a way to aggregate and manage capabilities under a human-meaningful identifier such as an email address.
+1. Instead of user agents trying discovering each to delegate capabilities with one another, all capabilities get delegated to an account which is then used to re-delegate them as necessary to individual agents.
+2. Recovery is possible even if all devices has been lost as long as the user retains control of their email, because account can always delegate capabilities to new agents.
 
-Specifically, we propose deriving an account identifier from a user-controlled email address in the form of a [`did:mailto`] identifier, which can act as the [principal] in [UCAN] delegation chains.
+Agent authorization can use familiar email-based authorization flows providing a smooth onboarding experience and hide complexity of the underlying [PKI]. This approach also better reflects user intuition: they have ambient authority over owned spaces and can authorize user agents (think apps) giving them necessary level of access.
 
-Using an account identifier based on a memorable email address solves the discovery problem, and email-based authorization flows provide a smoother onboarding experience by hiding the complexity of [PKI]. With this approach, users can aggregate all of their delegations under a single account identifier and re-delegate desired capabilities to other agents.
+> ℹ️ This specification mainly focuses on [`did:mailto`] identifiers, but implementations are free to extend it to various other [DID methods].
+>
+> Also, note that [account] is defined as non [`did:key`] identifier because [`did:key`] identifiers can aggregate and re-delegate capabilities natively with [UCAN]s. [Account]s simply bring human-meaningful identifiers that users can type in on every agent. Use of out of bound authorization also frees users from key management.
 
-> ℹ️ This specification mainly focuses on [`did:mailto`] identifiers, but it can be extended to various other types of identifiers.
+## Serialization
 
-# High-Level Concepts
+[UCAN]s MUST be encoded with some [IPLD] codec. [DAG-CBOR] is RECOMMENDED.
+
+## Concepts
 
 ## Roles
 
 There are several distinct roles that [principals] may assume in described specification:
 
 | Name        | Description                                                                                                                                    |
 | ----------- | ---------------------------------------------------------------------------------------------------------------------------------------------- |
+| Principal | The general class of entities that interact with a UCAN. Listed in the `iss` or `aud` field |
```

Comment:
most other terms in this table get elaborated upon in a `###` section below. It could be helpful to eventually do the same for Principal